adagrams/game.py

```diff
@@ -1,11 +1,141 @@
+import random
+from random import choices
+import copy
+
+LETTER_DICT = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
 def draw_letters():
-    pass
+
+    letter_bank = []
+    letter_copy = copy.deepcopy(LETTER_DICT)
+
+    while len(letter_bank) < 10:
+        key_list = list(letter_copy.keys())
+        random_letter = random.choice(key_list)
+        if letter_copy[random_letter] > 0:
+            letter_bank.append(random_letter)
+            letter_copy[random_letter] -= 1
+    print(letter_bank)
+    return letter_bank 
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+
+    case_word = word.upper()
+    letter_bank_copy = copy.deepcopy(letter_bank)
+    for letter in case_word:
+        if letter.upper() not in letter_bank_copy:
+            return False
+        elif letter.upper() in case_word:
+            letter_bank_copy.remove(letter)
+    return True
 
 def score_word(word):
-    pass
+    
+    SCORE_DICT = {
+    'A': 1, 
+    'B': 3, 
+    'C': 3, 
+    'D': 2, 
+    'E': 1, 
+    'F': 4, 
+    'G': 2, 
+    'H': 4, 
+    'I': 1, 
+    'J': 8, 
+    'K': 5, 
+    'L': 1, 
+    'M': 3, 
+    'N': 1, 
+    'O': 1, 
+    'P': 3, 
+    'Q': 10, 
+    'R': 1, 
+    'S': 1, 
+    'T': 1, 
+    'U': 1, 
+    'V': 4, 
+    'W': 4, 
+    'X': 8, 
+    'Y': 4, 
+    'Z': 10
+    }
+
+    case_word = word.upper()
+    sum_list = []
```

Comment:
You could simplify this by creating a sum variable that starts as an integer of 0 and then add the score based on the length of the word. Something like this:
```
def score_word(word):
    score = 0
    for letter in word:
        score += LETTER_VALUES[letter.upper()]

    if len(word) > 6 and len(word) < 11:
        score += 8

    return score
```